### PR TITLE
Reorg staking pages (intro to staking and advanced staking)

### LINKED ISSUE #3816: Reorg Staking pages (intro and advanced)
This issue follows [this closed PR](https://github.com/w3f/polkadot-wiki/pull/3787)

**Problem**
- [ ] Some minor fixes in rendering
- [ ] Diagrams with better graphics as suggested by @kianenigma

**Solution**
Fix rendering issues and propose some variants for the diagrams

### PR DESCRIPTION
**Pull request by filippoweb3**
This PR is a follow-up of two previous PRs about the staking pages. It is meant to address small fixes such as rendering issues and figures.

Previous second PR https://github.com/w3f/polkadot-wiki/pull/3787
Previous first PR https://github.com/w3f/polkadot-wiki/pull/3623

### CODE REVIEW

**Comment by filippoweb3** [comment]
Some versions of the wiki diagrams

Screenshot 2022-09-19 at 13 45 30

Screenshot 2022-09-19 at 13 54 16

Screenshot 2022-09-19 at 14 02 21

**Comment by filippoweb3** [comment]
@kianenigma this is the new staking PR. Let me know what you think about the proposed versions of the figures without the white background.

I tried to add Ross too, but I cannot find him!

File: docs/learn/learn-staking-advanced.md [hunk 1]
```diff
@@ -1,42 +1,48 @@
 ---
 id: learn-staking-advanced
 title: Advanced staking
-sidebar_label: Advanced staking
+sidebar_label: Staking
```
Comment (profradha) [inline docs/learn/learn-staking-advanced.md]: I see your reasoning for removing the word "Advanced", but it makes sense to keep it

```suggestion
sidebar_label: Advanced Staking Concepts
```

**Review by DrW3RK — APPROVED** [review]

File: docs/learn/learn-staking-advanced.md [hunk 2]
```diff
@@ -1,42 +1,48 @@
 ---
 id: learn-staking-advanced
 title: Advanced staking
```
Comment (profradha) [inline docs/learn/learn-staking-advanced.md]:
```suggestion
title: Advanced Staking Concepts
```

**Pull request merged**